.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
```

Comment:
LGTM. I am only concerned about having to manually upgrade the versions of these pre-commit hooks. 

I know Dependabot can upgrade GitHub actions but it looks like it doesn't support pre-commit hooks.

Reply:
The service does upgrades weekly, for example see https://github.com/asottile/pyupgrade/pull/570